server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -171,7 +171,8 @@ public GlobalStatus commit(String xid) throws TransactionException {
                 globalSession.asyncCommit();
                 return GlobalStatus.Committed;
             } else {
-                return globalSession.getStatus();
+                return globalSession.getStatus() == GlobalStatus.Committing ? GlobalStatus.Committed
+                    : globalSession.getStatus();
```

Comment:
if TCC mode, directly return Committing?

Reply:
yes,when the doGlobalCommit method is executed, the commtting status is equal to committed